config.example.js

```diff
@@ -8,5 +8,17 @@ module.exports = {
     // S3 Region
     s3Region: 'ap-southeast-2',
     // allow Mojito load more than one times
-    allowMultiInstance: false
+    allowMultiInstance: false,
+    // Lifecycle Events conf
+    lifecycleEvents: {
+        ci: {
+            branch: 'lifecycle-events-metadata-log'
+        },
+        analytics: {
+            snowplow: {
+                appId: 'mintmetrics',
+                collectorUrl: 'c.mintmetrics.io'
+            }
+        }
+    }
```

Comment:
Maybe we should remove these for the default config or use some placeholder values. @dapperdrop - thoughts on this?

For placeholder, perhaps use:

branch: `master`
appId: `your-app-id`
collectorUrl: `your.collector.com`


I'll need to prepare the docs on the config.

Reply:
yep that makes sense, not sure what the default mainline branch is between git platforms nowadays, so maybe better to use something like `your-mainline-branch` as a placeholder?

Reply:
True... maybe we need it to distinguish it as "your-production-branch" - the branch which we run the production pipeline over (and whether that's even different to the default branch).

Reply:
@allmywant can you please update this to

branch: your-production-branch
appId: your-app-id
collectorUrl: your.collector.com

Reply:
@kingo55 
Fixed.

Reply:
Thanks @allmywant - will get this merged now.